crates/admin/src/storage_query/query.rs

```diff
@@ -220,6 +228,7 @@ impl<W: RecordBatchWriter + Unpin> Stream for WriteRecordBatchStream<W> {
             match record_batch.and_then(|record_batch| self.stream_writer.write(&record_batch)) {
                 Ok(bytes) => Poll::Ready(Some(Ok(bytes))),
                 Err(err) => {
+                    warn!(query = %self.query, %err, "Query failed");
```

Comment:
The query is logged if the user is using --verbose i think

Reply:
Using `--verbose` where?